Tests/Marketplace/upload_packs.py

```diff
@@ -504,8 +504,11 @@ def main():
             pack.cleanup()
             continue
 
-        # todo finish implementation of release notes
-        # pack.parse_release_notes()
+        task_status = pack.prepare_release_notes(storage_bucket)
+        if not task_status:
+            pack.status = PackStatus.FAILED_RELEASE_NOTES.name
```

Comment:
If you planning to create `changelog.json` in that function, so the error message should be different.